packages/teleterm/src/services/workspace/workspaceService.ts

```diff
@@ -38,6 +39,24 @@ export function createWorkspaceService(options: {
       });
   }
 
+  function addToRecentDocuments(document: Document): void {
+    const maxRecentDocumentsCount = 5;
```

Comment:
no need for this limit, allow as many as it gets.

Reply:
Done